src/sql/magic.py

```diff
@@ -264,6 +278,17 @@ def _execute(self, payload, line, cell, local_ns):
         # args.line: contains the line after the magic with all options removed
 
         args = command.args
+        # Create the interactive slider
```

Comment:
we need to add a check here, if the user passed the interact argument, then we check that ipywidgets is installed using `check_installed`

Reply:
Handled in [e6b45dd](https://github.com/ploomber/jupysql/pull/293/commits/e6b45dd39b8b01856b8351407f346fe2ad583668)